kubernetes-tests/src/test/java/io/fabric8/openshift/client/server/mock/OpenShiftVersionInfoTest.java

```diff
@@ -46,7 +46,7 @@ void testClusterVersioningOpenshift3() throws ParseException {
     assertEquals("3", client.getVersion().getMajor());
     assertEquals("11+", client.getVersion().getMinor());
     assertEquals(119, client.getVersion().getBuildDate().getYear());
-    assertEquals(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'").parse("2019-10-31T21:06:55Z").getTime(), client.getVersion().getBuildDate().getTime());
+    assertEquals(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssX").parse("2019-10-31T21:06:55Z").getTime(), client.getVersion().getBuildDate().getTime());
```

Comment:
Curious about this change: has the date format changed?

Reply:
'Z' expects the literal character Z to appear in the date.  X will parse it as the timezone.  The logic was switched to using jackson for the deserialization, so it's now respecting the timezone.